docs/Maintainer-Guidelines.md

```diff
@@ -105,6 +107,28 @@ is a good opportunity to do it) provided the line itself has some kind
 of modification that is not whitespace in it. But be careful about
 making changes to inline patches—make sure they still apply.
 
+### Adding or update formulae
+Only one maintainer is necessary to approve and merge the addition of a new or updated formula which passes CI. However, if the formula addition or update is controversial the maintainer who adds it will be expected to fix issues that arise with it in future.
+
+### Removing formulae
+Formulae that:
+
+- work on at least 2/3 of our supported macOS versions in the default Homebrew prefix
+- do not require patches rejected by upstream to work
+- do not have known security vulnerabilities/CVEs for the version we package
+- are shown to be still installed by users in our analytics with a `BuildError` rate of <25%
+
+
+should not be removed from Homebrew. The exception to this rule are [versioned formulae](Versions.md) for which there are higher standards of usage and a maximum number of versions for a given formula.
+
+### Closing issues/PRs
+Maintainers (including the lead maintainer) should not close issues or pull requests opened by other maintainers unless they are stale (i.e. have seen no updates for 28 days) in which case they can be closed by any maintainer. Any maintainer is encouraged to reopen a closed issue when they wish to do additional work on the issue.
```

Comment:
Since merging also closes an issue this could be read as "if I submit a pr only I can merge it" perhaps using the title `Closing issues/PRs (except merges)` would just emphasise that even if its written above.

Reply:
@javian Good idea!